src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Data/Generated/Models/SuggestParameters.cs

```diff
@@ -0,0 +1,170 @@
+        /// query will find terms even if there's a substituted or missing
+        /// character in the search text. While this provides a better
+        /// experience in some scenarios it comes at a performance cost as
+        /// fuzzy autocomplete queries are slower and consume more resources.
+        /// </summary>
+        [Newtonsoft.Json.JsonIgnore]
+        public bool UseFuzzyMatching { get; set; }
+
+        /// <summary>
+        /// Gets or sets a string tag that is appended to hit highlights. Must
+        /// be set with HighlightPreTag. Default is "".
+        /// </summary>
+        [Newtonsoft.Json.JsonIgnore]
+        public string HighlightPostTag { get; set; }
+
+        /// <summary>
+        /// Gets or sets a string tag that is prepended to hit highlights. Must
+        /// be set with HighlightPostTag. Default is "".
+        /// </summary>
+        [Newtonsoft.Json.JsonIgnore]
+        public string HighlightPreTag { get; set; }
+
+        /// <summary>
+        /// Gets or sets a number between 0 and 100 indicating the percentage
+        /// of the index that must be covered by a suggestions query in order
+        /// for the query to be reported as a success. This parameter can be
+        /// useful for ensuring search availability even for services with only
+        /// one replica. The default is 80.
+        /// </summary>
+        [Newtonsoft.Json.JsonIgnore]
+        public double? MinimumCoverage { get; set; }
+
+        /// <summary>
+        /// Gets or sets the comma-separated list of OData $orderby expressions
+        /// by which to sort the results. Each expression can be either a field
+        /// name or a call to either the geo.distance() or the search.score()
+        /// functions. Each expression can be followed by asc to indicate
+        /// ascending, and desc to indicate descending. The default is
+        /// ascending order. Ties will be broken by the match scores of
+        /// documents. If no OrderBy is specified, the default sort order is
+        /// descending by document match score. There can be at most 32 Orderby
+        /// clauses.
+        /// </summary>
+        [Newtonsoft.Json.JsonIgnore]
+        public IList<string> OrderBy { get; set; }
+
+        /// <summary>
+        /// Gets or sets the list of comma-separated field names to search for
+        /// the specified search text. Target fields must be enabled for
+        /// suggestions.
+        /// </summary>
+        [Newtonsoft.Json.JsonIgnore]
+        public IList<string> SearchFields { get; set; }
+
+        /// <summary>
+        /// Gets or sets the comma-separated list of fields to retrieve. If
+        /// unspecified, all fields marked as retrievable in the schema are
+        /// included.
+        /// </summary>
+        [Newtonsoft.Json.JsonIgnore]
+        protected internal string SelectStr { get { return (Select != null && Select.Any()) ? string.Join(",", Select) : "*"; } set { } }
```

Comment:
I don't like this either. IIRC, this is due to the discrepancy in behavior of unspecified select in Rest API and SDK. In rest API, unspecified $select means [no select](https://docs.microsoft.com/en-us/rest/api/searchservice/suggestions). While in SDK, unspecified select means '*'. SelectStr is what I came up to preserve the backward compatibility/discrepancy, but I think it will make our life easier if we just make the two consistent before we expand to other languages.